src/DFC_BusinessOntology.owl

```diff
@@ -5904,9 +5454,17 @@
     <ObjectPropertyDomain>
         <ObjectProperty IRI="#hasQuantity"/>
         <ObjectUnionOf>
+            <Class IRI="#ConsumptionFlow"/>
             <Class IRI="#DefinedProduct"/>
             <Class IRI="#Ingredient"/>
```

Comment:
Looks good. 

Just noting Ingredient wasn't ever linked to the deprecated `#quantity` DataProperty. Is it too new ? 🤨

Reply:
No, but there were many problems with the Ingredient class because it disappeared during several update. Maybe I recreated the Ingredient class directly with the right property